# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/server/actors/replay/replay.js
# Commit: 73c33bf110d6
# Full Hash: 73c33bf110d6034266cbe615b9e018476a128d63
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 2
# Description:
#   Bug 1600204 - Part 2: Remove Debugger.Frame.prototype.generator since it is already on script. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55232
# ==============================================================================

diff -r d70ec1e79887 -r 73c33bf110d6 devtools/server/actors/replay/replay.js
--- a/devtools/server/actors/replay/replay.js	Fri Dec 06 02:29:18 2019 +0000
+++ b/devtools/server/actors/replay/replay.js	Fri Dec 06 02:57:20 2019 +0000
@@ -1350,7 +1350,6 @@
     type: frame.type,
     callee: getObjectId(frame.callee),
     environment: getObjectId(frame.environment),
-    generator: frame.generator,
     constructing: frame.constructing,
     this: convertValue(frame.this),
     script,